java/src/main/java/com/mlt/converter/triangulation/TriangulatedPolygon.java

```diff
@@ -0,0 +1,23 @@
+package com.mlt.converter.triangulation;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TriangulatedPolygon {
```

Comment:
If I'm reading build.gradle correctly, it looks like we're compiling against java 17. If that's right could this class just be `record TriangulatedPolygon(List<Integer> indexBuffer, int numTrianglesPerPolygon) {}` ?